Base/Python/slicer/util.py

```diff
@@ -140,29 +140,20 @@ def findChildren(widget=None,name="",text="",title="",className=""):
     if not hasattr(p,'children'):
       continue
     parents += p.children()
-    if name and fnmatch.fnmatchcase(p.name, name):
-      children.append(p)
-    elif text:
-      try:
-        p.text
-        if fnmatch.fnmatchcase(p.text, text):
-          children.append(p)
-      except (AttributeError, TypeError):
-        pass
-    elif title:
-      try:
-        p.title
-        if fnmatch.fnmatchcase(p.title, title):
-          children.append(p)
-      except AttributeError:
-        pass
-    elif className:
-      try:
-        p.className()
-        if fnmatch.fnmatchcase(p.className(), className):
-          children.append(p)
-      except AttributeError:
-        pass
+    condition = True
+    if name:
+      condition &= fnmatch.fnmatchcase(p.name, name)
```

Comment:
If p doesn't have a name attribute then an exception is raised and all further children are ignored. To fix this, you would need to change
fnmatch.fnmatchcase(p.name, name) 
to something like
hasattr(p, 'name') and fnmatch.fnmatchcase(p.name, name)
for all checks (name, text, title, className)